custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js

```diff
@@ -79,6 +83,29 @@ function MapOrSectorController($location, storageService, locationsService) {
                         }
                     });
                 });
+
+                d3.selectAll(".nv-x.nv-axis .tick text").each(function(i, e) {
+                    var text = d3.select(this),
```

Comment:
I think it would be helpful if this function were named to explain what it's doing (I think something about wrapping text inside SVGs?) and to the extent that it was taken from SO (I think [at least somewhat](https://stackoverflow.com/a/28553412/8207)) it's always good to cite so someone can understand/follow up at a later date

Reply:
@czue I changed this